apis/server/container_bridge.go

```diff
@@ -186,6 +187,24 @@ func (s *Server) getContainers(ctx context.Context, rw http.ResponseWriter, req
 	return EncodeResponse(rw, http.StatusOK, containerList)
 }
 
+func (s *Server) killContainer(ctx context.Context, rw http.ResponseWriter, req *http.Request) error {
+	var sig syscall.Signal
+	name := mux.Vars(req)["name"]
+
+	// If we have a signal, look at it. Otherwise, do nothing
+	if sigStr := req.FormValue("signal"); sigStr != "" {
+		var err error
+		if sig, err = utils.ParseSignal(sigStr); err != nil {
+			return err
+		}
+	}
+
+	if err := s.ContainerMgr.Kill(ctx, name, uint64(sig)); err != nil {
```

Comment:
please just `return s.ContainerMgr.Kill(ctx, name, uint64(sig))` to simplify code.